developer_docs/how_to_solve_an_issue.md

```diff
@@ -15,6 +15,6 @@ To add a new function:
 ## Language Specific Bug Fixes
 
 -   Use the issue description to reproduce the bug
--   Add a test targeting this specific problem (which fails in the master branch but will pass after your fix)
+-   Add a test targeting this specific problem (which fails in the devel branch but will pass after your fix)
```

Comment:
Should this be

```suggestion
-   Add a test targeting this specific problem (which fails in the `devel` branch but will pass after your fix)
```
Rather than adding `devel` to the dictionary?

Reply:
You may be right. There were 8 separate instances which triggered the failure in the spell checker, so I assumed that we needed to add the word to the dictionary...